Makefile

```diff
@@ -15,20 +15,19 @@ coverage_xml:
 check: black ruff test coverage_xml
 
 install:
-	python -m pip install -e .
+	python -m pip install uv
+	uv sync
 
 install-dev:
 	pip install --upgrade pip setuptools wheel
-	python -m pip install -e ".[dev]"
+	python -m pip install uv
+	uv sync --extra "dev"
 	pre-commit install
 
 install-test:
 	pip install --upgrade pip setuptools wheel
-	python -m pip install -e ".[test]"
-
-install-only-test:
```

Comment:
@david26694 Is this needed? Seems weird to me and is a bit weird doing it in toml file

Reply:
not really